xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -0,0 +1,1149 @@
+          <emphasis>final</emphasis>:</para>
+
+          <para><screen>workflow:
+  steps:
+    - branch_package:
+      source_project: home:jane_doe
+      source_package: ctris
+      target_project: games
+  filters:
+    branches:
+      ignore:
+        - master
+        - final </screen></para>
+
+          <para>Learn more about <link
+          linkend="sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows.filters.delimiters">Filters
+          Delimiters: only and ignore</link>.</para>
+        </sect4>
+
+        <sect4>
+          <title>Architectures Filter</title>
+
+          <para>This filter matches architectures based on their names and
+          report back to GitHub only for those architectures.</para>
+
+          <para>Example of a workflow reporting back to the GitHub only for
+          architectures <emphasis>s390x</emphasis> and
+          <emphasis>x86_64</emphasis>:</para>
+
+          <para><screen>workflow:
+  steps:
+    - branch_package:
+      source_project: home:jane_doe
+      source_package: ctris
+      target_project: games
+  filters:
+    architectures:
+      only:
+        - s390x
+        - x86_64</screen></para>
+
+          <para>Learn more about <link
+          linkend="sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows.filters.delimiters">Filters
+          Delimiters: only and ignore</link>.</para>
+        </sect4>
+
+        <sect4>
+          <title>Repositories Filter</title>
+
+          <para>Matches repositories based on their names and report back to
+          GitHub only for those repositories.</para>
+
+          <para>Example of a workflow reporting back to GitHub only for
+          repositories <emphasis>openSUSE_Tumbleweed</emphasis> and
+          <emphasis>CentOS_8</emphasis>:</para>
+
+          <para><screen>workflow:
+  steps:
+    - branch_package:
+      source_project: home:jane_doe
+      source_package: ctris
+      target_project: games
+  filters:
+    repositories:
+      only:
+        - openSUSE_Tumbleweed
+        - CentOS_8</screen></para>
+
+          <para>Learn more about <link
+          linkend="sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows.filters.delimiters">Filters
+          Delimiters: only and ignore</link>.</para>
+        </sect4>
+      </sect3>
+    </sect2>
+
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.status_reporting">
+      <title>Status Reporting</title>
+
+      <para>Once all the steps in the workflow are done, the OBS will have to report the
+      results back to GitHub.</para>
+
+      <para>OBS will show detailed package build status for each distribution
+      and architecture you have set up in the configuration file.</para>
+
+      <mediaobject>
+        <imageobject>
+          <imagedata fileref="https://openbuildservice.org/images/posts/blog-post-about-reporting-repositories-architectures-and-reacting-to-updates-in-a-pr/commit_status.png"/>
```

Comment:
Same here, screenshot is to wide on my generated PDF

Reply:
Done! I've fixed the image's width and they are correctly displayed in the PDF version too.